cvat/apps/dataset_manager/util.py

```diff
@@ -160,14 +160,20 @@ def get_export_cache_dir(db_instance: Project | Task | Job) -> str:
 def make_export_filename(
     dst_dir: str,
     save_images: bool,
+    all_images: bool,
     instance_timestamp: float,
     format_name: str,
 ) -> str:
     from .formats.registry import EXPORT_FORMATS
     file_ext = EXPORT_FORMATS[format_name].EXT
 
+    if save_images and all_images:
+        dataset_type = 'fulldataset'
+    else:
+        dataset_type = 'dataset' if save_images else 'annotations'
```

Comment:
It will be better to keep compatibility with the current behavior, and make options additional to the default behavior
```suggestion
    dataset_type = 'dataset' if save_images else 'annotations'
    if not all_images:
        dataset_type += "-annotated_only"
```